fluent-plugin-logdna.gemspec

```diff
@@ -3,7 +3,7 @@ require 'date'
 
 Gem::Specification.new do |s|
   s.name        = 'fluent-plugin-logdna'
-  s.version     = '0.1.7.1'
+  s.version     = '0.1.9.1'
```

Comment:
We are adding a new feature - tag support; I think because of that we need to bump up minor version and be loyal to SemVer norms; in other words, instead of `0.1.9.1`,  `0.2.0` would be suitable.

Reply:
Good call!